snap/snapcraft.yaml.in

```diff
@@ -0,0 +1,263 @@
+name: kata-containers
+version: @KATA_RUNTIME_VERSION@
+summary: Build lightweight VMs that seamlessly plug into the containers ecosystem
+description: |
+  Kata Containers is an open source project and community working to build a
+  standard implementation of lightweight Virtual Machines (VMs) that feel and
+  perform like containers, but provide the workload isolation and security
+  advantages of VMs
+grade: stable
+confinement: classic
+
+parts:
+  go:
+    source-tag: go@GO_VERSION@
+  runtime:
+    source: http://github.com/kata-containers/runtime
+    source-type: git
+    source-tag: @KATA_RUNTIME_VERSION@
+    after: [go]
+    plugin: go
+    go-importpath: github.com/kata-containers/runtime
+    build-attributes: [no-patchelf]
+    override-build: |
+      # Don't use installed GOROOT
+      unset GOROOT
+      export GOPATH=$(realpath ../go)
+      cd ${GOPATH}/src/github.com/${SNAPCRAFT_PROJECT_NAME}/runtime
+      make \
+        QEMUPATH=/snap/${SNAPCRAFT_PROJECT_NAME}/current/usr/bin/qemu-system-x86_64 \
```

Comment:
Just a note for our other arch's - this is hard wired to x86 - I presume there is snap support for other arch's - so we'll need to update/clone this when we get to supporting them via snaps.